### PR TITLE
update tutorial.rst

### PR DESCRIPTION
**Pull request by fakesecret**
**Proposed changes**: update tutorial.rst
- I hope the tutorial gets a little clearer for beginners now.

- There is still room for improvement, especially regarding how to prepare the training data. For non-programmers, rasa-nlu-trainer is a good choice; for informaticiens, since there are no scripts of generating, analyzing, merging training data provided by rasa-NLU (at least,  not mentionned in the docs), they need to create these scripts if they want to deal with more training data with more complex intent/entities models.

- Maybe a few pieces of advices/rules can be also added to explain how to design good intent/entities models.

### CODE REVIEW

**Comment by wrathagom** [comment]
Excellent changes, any chance you want to include a bit in the `Train your Model` section on training via http?

If not I might add it in later. I need to write a good docker tutorial anyway.

File: docs/tutorial.rst [hunk 1]
```diff
@@ -28,18 +28,17 @@ And even more ways to say that you want to look for restaurants:
...
 
 Preparing the Training Data
 ---------------------------
 
-The best way to get training data is from *real users*, and the best way to do that is to `pretend to be the bot yourself <https://conversations.golastmile.com/put-on-your-robot-costume-and-be-the-minimum-viable-bot-yourself-3e48a5a59308#.d4tmdan68>`_. But to help get you started we have some `data saved <https://github.com/RasaHQ/rasa_nlu/blob/master/data/examples/rasa/demo-rasa.json>`_
-
-Download the file and open it, and you'll see a list of training examples like these:
+The training data are essential to develop chatbots. They should include texts to be interprated and the structured data (intent/entities) we expect chatbots to convert the texts into. The best way to get training texts is from *real users*, and the best way to get the structured data is to `pretend to be the bot yourself <https://conversations.golastmile.com/put-on-your-robot-costume-and-be-the-minimum-viable-bot-yourself-3e48a5a59308#.d4tmdan68>`_. But to help get you started, we have some `data saved <https://github.com/RasaHQ/rasa_nlu/blob/master/data/examples/rasa/demo-rasa.json>`_.
```
Comment (tmbo) [inline docs/tutorial.rst]: The training data **is** ... (we treat the word data as a singular term, let's keep it that way)
... to be **interpreted** ...

File: docs/tutorial.rst [hunk 2]
```diff
@@ -212,4 +209,4 @@ With very little data, rasa NLU can in certain cases already generalise concepts
     }
 
 even though there's nothing quite like this sentence in the examples used to train the model. 
-To build a more robust app you will obviously want to use a lot more data, so go and collect it!
+To build a more robust app you will obviously want to use a lot more training data, so go and collect them!
```
Comment (tmbo) [inline docs/tutorial.rst]: let's keep the "it" (I know there is a heated debate about it - but let's stay consistent across the documentation)

File: docs/tutorial.rst (hunk 1 above)
Comment (fakesecret) [inline docs/tutorial.rst]: Sorry about the typo. Regarding "data", when I publish articles in the bioinformatics field, I think we can consider this word either a plural form of datum or an uncountable noun meaning information, depending on contexts. So, the both for me are good, but I agree with and respect your choice.

**Comment by fakesecret** [comment]
@tmbo @wrathagom I want to change another important place in this file and tell me what you think about.
"Training your model": $ python -m rasa_nlu.train -c sample_configs/**config_spacy.json**
"Using your model": $ python -m rasa_nlu.server -c sample_configs/**config_spacy.json** --server_model_dirs=./model_YYYYMMDD-HHMMSS
However, the results shown were not spaCy results, because so few examples would not result in any extracted entities. I remember, this step misguided me and it took me one hour to figure out whether I had done something wrong, until I read the following sentence of "do not panic!". Are you using Mitie here or another custom pipeline? It is very important to make them consistent here for beginners.

**Comment by fakesecret** [comment]
@wrathagom How to train via http? Do you mean preparing/modifying training data using rasa-nlu-trainer? I think a figure that shows the fields for modification in addition to the figure rasa_nlu_intent_gui.png will be a good start. I also see some improvements to do for **Training Data Format** page.

In terms of Docker, I haven't used it for now, but one NLP engineer friend told me he had some troubles in using that.

**Pull request merged**